_style/index.md

```diff
@@ -69,6 +69,4 @@ This document is intended to outline some basic Scala stylistic guidelines which
     - [Traits](scaladoc.html#traits)
   - [Methods and Other Members](scaladoc.html#methods-and-other-members)
 
-### Thanks to ###
-
-[Daniel Spiewak](https://www.codecommit.com/) and [David Copeland](https://www.naildrivin5.com/) for putting this style guide together, and Simon Ochsenreither for converting it to Markdown.
+**Thanks to** [Daniel Spiewak](https://www.codecommit.com/) and [David Copeland](https://www.naildrivin5.com/) for putting this style guide together, and Simon Ochsenreither for converting it to Markdown.
```

Comment:
I think enough years have now passed, and enough people have contributed, that it's no longer necessary to have a "Thanks to" section at all.